tests/modules/usercall.c

```diff
@@ -1,4 +1,8 @@
 #include "redismodule.h"
+#include <pthread.h>
+#include <assert.h>
+
+#define UNUSED(V) ((void) V)
```

Comment:
REDISMODULE_UNUSED I think comes from redismodule.h

Reply:
thanks.. apparently all our test modules do this (use `UNUSED`).
but anyway, does this comment means you have nothing else to comment on this PR?
specifically i really need feedback about the change for CSC and scripts.
do you know who else (other than Salvatore) can comment on that?

Reply:
No, I didn't do a complete look yet.